src/main/lighting/AmbientLight.java

```diff
@@ -0,0 +1,25 @@
+package lighting;
+
+import primitives.Colour;
+
+/**
+ * This light models the small ammounts of light which reflected so many times that their source is indeterminable. All
```

Comment:
*amount. And that's not really what ambient light is.

Reply:
![Screenshot_20210504-235326.jpg](https://user-images.githubusercontent.com/25041135/117068480-3d5e8f80-ad23-11eb-9617-58a4fd7d9c6b.jpg)

That's where I got it from. If you prefer another explanation, feel free to change it.